.travis.yml

```diff
@@ -11,10 +11,23 @@ env:
 - PACKAGES="Django>=1.9,<1.10 django-threadedcomments>=1.1"
 - PACKAGES="Django>=1.10,<1.11"
 - PACKAGES="Django>=1.10,<1.11 django-threadedcomments>=1.1"
+- PACKAGES="Django>=1.11,<1.12"
+- PACKAGES="Django>=1.11,<1.12 django-threadedcomments>=1.1"
+- PACKAGES="--pre Django>=2.0a1,<2.1"
+- PACKAGES="--pre Django>=2.0a1,<2.1 django-threadedcomments>=1.1"
 - PACKAGES='https://github.com/django/django/archive/master.tar.gz'
 matrix:
+  exclude:
+  - python: '2.7'
+    env: PACKAGES="--pre Django>=2.0a1,<2.1"
+  - python: '2.7'
+    env: PACKAGES="--pre Django>=2.0a1,<2.1 django-threadedcomments>=1.1"
+  - python: '2.7'
+    env: PACKAGES='https://github.com/django/django/archive/master.tar.gz'
   allow_failures:
   - env: PACKAGES='https://github.com/django/django/archive/master.tar.gz'
+  - python: '3.5'
+    env: PACKAGES="--pre Django>=2.0a1,<2.1 django-threadedcomments>=1.1"
```

Comment:
this can use rc1 now :-)

Reply:
It should use the latest version available. But we can wait for official release and write `2.0`.

Reply:
I updated this to plain Django 2.0 now.